apps/main/src/lend/hooks/useBandsData.ts

```diff
@@ -0,0 +1,40 @@
+import { PageContentProps } from '@/lend/types/lend.types'
+import { useProcessedBandsData } from '@/llamalend/features/bands-chart/hooks/useProcessedBandsData'
+import { useMarketBands } from '@/llamalend/features/bands-chart/queries/market-bands.query'
+import { useMarketOraclePrices } from '@/llamalend/features/bands-chart/queries/market-oracle-prices.query'
+import { useUserBands } from '@/llamalend/features/bands-chart/queries/user-bands.query'
+import { useLoanExists } from '@/llamalend/queries/loan-exists'
+
+export const useBandsData = ({ rChainId, rOwmId, api }: Pick<PageContentProps, 'rChainId' | 'api' | 'rOwmId'>) => {
+  const { data: marketOraclePrices, isLoading: isMarketOraclePricesLoading } = useMarketOraclePrices({
+    chainId: rChainId,
+    marketId: rOwmId,
+  })
+  const { data: marketBands, isLoading: isMarketBandsLoading } = useMarketBands({ chainId: rChainId, marketId: rOwmId })
+  const { data: loanExists, isLoading: isLoanExistsLoading } = useLoanExists({
+    chainId: rChainId,
+    marketId: rOwmId,
+    userAddress: api?.signerAddress,
```

Comment:
FYI the wallet address returned by useConnection might have the signer address before the `api` is initialized, which might speed up things a little bit